src/components/utils/text-field-with-adornment.js

```diff
@@ -41,18 +39,15 @@ const TextFieldWithAdornment = (props) => {
     } = props;
     const [isFocused, setIsFocused] = useState(false);
 
-    const getAdornmentClassName = useCallback(
-        (variant) => {
-            if (variant === 'filled') {
-                return classes.adornRightFilled;
-            }
-            if (variant === 'standard') {
-                return classes.adornRightOther;
-            }
-            return null;
-        },
-        [classes.adornRightFilled, classes.adornRightOther]
-    );
+    const getAdornmentStyle = useCallback((variant) => {
```

Comment:
You took the same code I did in commons ui ? To be sure there is no differences

Reply:
I did not, do we need this component here if we have it in commons-ui ?

Reply:
No we won't need it anymore, but my PR is not merged yet

Reply:
component removed after commons-ui update